### ISSUE TITLE
Setting to offset the bottom part

### ISSUE THREAD

**Issue by bsas** [League-of-Foundry-Developers/fvtt-module-theatre#14]
So, if you have the hotkey bar or the users list active the Narrator (Stage) shows bellow all that.
If would be nice to have a setting to configure how many pixels to offset the entire thing so one can move it up and avoid the issue.
A simple setting with "Offset top: 0px (default)" it would be sufficient I think.

**Comment by russellaugust** [comment]
yup, if you're using this for just showing who's speaking an not using the text option, the image is too low
